src/main/java/duke/parser/Parser.java

```diff
@@ -0,0 +1,70 @@
+package duke.parser;
+
+import duke.command.*;
+
+/**
+ * Represents a parser that makes sense of the command provided by the user.
+ * A <code>ParserTest</code> corresponds to a parsing tool whose method can be
+ * used universally to make sense of a user input.
+ */
+public class Parser {
+
+    /**
+     * Returns the command based on the input of the user. If the command is
+     * not valid, it returns a <code>HelpCommand</code>, else it returns one
+     * that has specific execution instructions that prompt and guide the bot
+     * towards specific execution, actions, and behaviours..
+     *
+     * @param fullCommand The input provided by the user using the CLI.
+     * @return the specific command generated by the user input.
+     */
+    public static Commands parse(String fullCommand) {
+        StringBuilder arguments = new StringBuilder("");
+        fullCommand = fullCommand.trim();
+        String commandWord = fullCommand.split(" ", 2)[0].trim();
+        if (fullCommand.split(" ", 2).length > 1) {
+            arguments.append(fullCommand.split(" ", 2)[1].trim());
+        }
+
+        switch (commandWord) {
+
```

Comment:
This is me being picky again, but maybe you could remove some of the blank lines to increase readability? However, in terms of code, everything looks good for the switch case.